blog/2019-06-03-debezium-newsletter-01-2019.adoc

```diff
@@ -0,0 +1,48 @@
+= Debezium's Newsletter 01/2019
+ccranfor
+:awestruct-tags: [ community, news, newsletter ]
+:awestruct-layout: blog-post
+
+Welcome to the Debezium community newsletter in which we share blog posts, group discussions, as well as StackOverflow
+questions that are relevant to our user community.
+
+== Articles
```

Comment:
Another recent and really good one is this: https://medium.com/@sincysebastian/setup-kafka-with-debezium-using-strimzi-in-kubernetes-efd494642585

And the recording from my Kafka Summit session might be worth sharing, too: https://www.confluent.io/kafka-summit-lon19/change-data-streaming-patterns-microservices-debezium